### ISSUE TITLE
latin-1 codec can't encode character

### ISSUE THREAD

**Issue by rodrigob** [sdg-mit/gitless#33]
```
# Oops...something went wrong (recall that Gitless is in beta). If you want to help, report the bug at http://gitless.com/community.html and include the following information:
    
    0.6.2
    
    Traceback (most recent call last):
      File "/home/benenson/.local/lib/python3.2/site-packages/gitless/cli/gl.py", line 73, in main
        return SUCCESS if args.func(args) else ERRORS_FOUND
      File "/home/benenson/.local/lib/python3.2/site-packages/gitless/cli/gl_history.py", line 41, in main
        puts(l, stream=tf.write)
      File "/home/benenson/.local/lib/python3.2/site-packages/clint/textui/core.py", line 68, in puts
        stream(_str)
    UnicodeEncodeError: 'latin-1' codec can't encode character '\u2019' in position 9: ordinal not in range(256)
```

**Comment by spderosso** [comment]
57259b7eddc940a6ec1fb02b312cd14fffd6d0ff

**Comment by spderosso** [comment]
Fix will go out in next release (0.7)
